crates/uv/src/cli.rs

```diff
@@ -1947,6 +1956,68 @@ pub(crate) struct AddArgs {
     /// - `/home/ferris/.local/bin/python3.10` uses the exact Python at the given path.
     #[arg(long, short, env = "UV_PYTHON", verbatim_doc_comment)]
     pub(crate) python: Option<String>,
+
+    /// The strategy to use when resolving against multiple index URLs.
+    ///
+    /// By default, `uv` will stop at the first index on which a given package is available, and
+    /// limit resolutions to those present on that first index (`first-match`. This prevents
+    /// "dependency confusion" attacks, whereby an attack can upload a malicious package under the
+    /// same name to a secondary
+    #[arg(long, value_enum, env = "UV_INDEX_STRATEGY")]
+    pub(crate) index_strategy: Option<IndexStrategy>,
```

Comment:
I think you could feasibly omit this, `keyring_provider`, `python_version`, `python_platform`, `link_mode`, `build_isolation`, ... We don't support them in `lock` or `sync` or `run` yet and need to tackle the problem holistically. Honestly, it might be _best_ to omit them for now so we don't tie ourselves to them.